x-pack/plugins/observability_solution/synthetics/server/synthetics_service/project_monitor/normalizers/common_fields.test.ts

```diff
@@ -8,9 +8,24 @@
 import {
   flattenAndFormatObject,
   getNormalizeCommonFields,
+  getUrlsField,
   NormalizedProjectProps,
 } from './common_fields';
 
+describe('getUrlsField', () => {
+  it('supports a string value containing a comma', () => {
+    expect(getUrlsField('https://elastic.co?foo=bar,baz')).toEqual([
+      'https://elastic.co?foo=bar,baz',
+    ]);
+  });
+
+  it('supports lists containing exactly one entry, even with commas', () => {
+    expect(getUrlsField(['https://elastic.co?foo=bar,baz'])).toEqual([
+      'https://elastic.co?foo=bar,baz',
+    ]);
```

Comment:
maybe a simple test for an invalid url as well?

Reply:
https://github.com/elastic/kibana/pull/186112/commits/f74d0bcf1c78539520c495a2b464f5f284e58894